Pipfile

```diff
@@ -20,6 +20,7 @@ responses = "*"
 pytest-cov = "*"
 functions-framework = "*"
 python-dotenv = "*"
+types-requests = "*"
```

Comment:
Not sure why this has been added?

Reply:
It was needed for the type hinting which suggested I install it since requests lib is not typed by default, so we have to load in extra stubs.

Reply:
I just looked at the card and the PR, which didn't mention type hinting

Reply:
As in `mypy` which does the type hinting. `mypy` got updated as part of this and it now errors on requests without its type stubs.